specs/signature-specification.md

```diff
@@ -95,13 +29,25 @@ The Notary Project signature supports the following envelope formats:
 
 The Notary Project signature payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
 
-- `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed. Only [OCI descriptor][oci-descriptor] is supported.
+- `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed.
+- For OCI artifacts, this MUST be a valid [OCI descriptor][oci-descriptor].
   - Descriptor MUST contain `mediaType`, `digest`, and `size` fields.
-  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. The Notary Project signature uses annotations for storing both Notary Project specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary Project signature and MUST NOT be used outside this specification.
+  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules].
   - Descriptor MAY contain `artifactType` field for the `config.mediaType` of OCI image manifests.
+- For Blob artifacts, the descriptor MUST describe the blob that is being signed
+  - Descriptor MUST contain `mediaType`, `digest`, and `size` fields.
+  - `digest` MUST be in the format of `<digest algorithm>:<digest value>`. Example: `sha256:2f3a23b6373afb134ddcd864be8e037e34a662d090d33ee849471ff73c873345`
```

Comment:
nit: we need to capture the constraint on this digest (like accepted values and how they are derived) some here.